source/includes/_tipser-elements.md

```diff
@@ -157,7 +139,11 @@ onError | Main goal of this handler is to add additional behavior when the error
     message?: string;
     stack?: string;
 }```
-onAddToCart | when product is being added to cart event is triggered | object of type {cartSize, product} where cartSize is a current size of cart after adding to cart and product is a product object with properties ```js 
+onAddToCart | when product is being added to cart event is triggered | object of type {cartSize, product} where cartSize is a current size of cart after adding to cart and product is a product object with properties see (see [TipserProductModel](#TipserProductModel-interface)) 
```

Comment:
same must be done for `EventError` interface on `onError` handler, because currently it looks like this:

<img width="899" alt="Zrzut ekranu 2019-05-9 o 15 56 51" src="https://user-images.githubusercontent.com/3996881/57459250-25a6ba80-7273-11e9-9353-4443909d32a3.png">